src/commons/RestAPI.js

```diff
@@ -0,0 +1,77 @@
+/** eslint verified */
+import axios from 'axios';
+
+// Data mockups
+import { geometryDAGMA } from '../assets/mockups/summaryData';
+
+
+class RestAPI {
+  /**
+   * Request the user information
+   *
+   * @param {String} username name in database
+   * @param {String} password password in database
+   */
+
+  /** *************** */
+  /** MAPS IN SUMMARY */
+  /** *************** */
+
+  /**
+   * Request area geometry by id
+   *
+   * @param {String} areaId area id to request
+   */
+  static requestGeometryByArea(areaId) {
+    if (areaId === 'LLA 0970') {
+      /* TODO: Implement geometry request
+      const source = CancelToken.source();
+      return {
+        request: RestAPI.makeGetRequest(`${areaId}/geometry`, { cancelToken: source.token }),
+        source,
+      };
+    */
+      return geometryDAGMA;
+    } return geometryDAGMA;
```

Comment:
While the final behavior is implemented this should only return geometryDAGMA, all that commented code and useless if statement is confusing and can be left there by mistake later. (Besides, we already have that code for reference in another repo)